front/public/locales/en/errors.json

```
@@ -238,6 +241,7 @@
     "study": {
       "Database": "Internal error (database)",
       "NotFound": "Study '{{study_id}}' could not be found",
+      "ProjectNotFound": "Project '{{project_id}}' could not be found",
```

nit: To start harmonizing with other translations ?
```suggestion
      "ProjectNotFound": "Project '{{project_id}}' not found",
```